### DISCUSSION TITLE
Difference between Environment variables and Variables

### DISCUSSION THREAD

**Discussion by weeping-somnambulist** [go-task/task#509] [Q&A]
Whats the functional difference between environment variables and regular variables here? Are environment variables exposed to the command while variables are not and are only used by the Task binary?

**Comment by andreynering** [comment]
Hi @SwitchedToGitlab,

Yep, that's about it. Environment variables can also be used as variables (for convenience), but their main intention to be passed to commands as env. Variables, on the other way, are primarily intended for interpolation.
